### PR TITLE
Use workflow instead of order_type

### PR DESCRIPTION
**Pull request by islean**
## Description

Currently order entries are created using order type as workflow. This is not quite correct since for example sars-cov-2 orders should have mutant as workflow but sars-cov-2 as order type.

### Fixed

- Use workflow instead of order type when creating orders.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/store/crud/create.py [hunk 1]
```diff
@@ -409,10 +409,11 @@ def add_sample_lane_sequencing_metrics(
 
     def add_order(self, order_data: OrderIn):
         customer: Customer = self.get_customer_by_internal_id(order_data.customer)
+        workflow: str = order_data.samples[0].data_analysis
```
Comment (seallard) [inline cg/store/crud/create.py:412]: Can we be sure that samples exist and that the data analysis is set on them?

**Review by seallard — APPROVED** [review]
Should work 👍 But not 100 % we can always assume the data analysis is available this way

File: cg/store/crud/create.py (hunk 1 above)
Comment (islean) [reply cg/store/crud/create.py:412]: I interpret this as us having minimum one sample: 
image

and data_analysis is not nullable in the basemodel 
image


Maybe @henrikstranneheim can confirm that we can not have an order without samples, and that we can not have samples without data_analysis set?

Comment (karlnyr) [reply cg/store/crud/create.py:412]: We can always assume that all orders should have 1 sample, and that sample should always have a workflow. What I do not know is whether or not we validate this. But we probably do 👍

Comment (islean) [reply cg/store/crud/create.py:412]: Yeah, I have seen this validation at least in the new order portal. And if not, I would guess we would want to throw an error.

**Comment by islean** [comment]
Tested on stage and now sars-covid orders are submitted with mutant as workflow.

**Comment by islean** [comment]
Deployed to the VMs.

**Pull request merged**